pkg/types/nutanix/machinepool.go

```diff
@@ -6,13 +6,13 @@ type MachinePool struct {
 	// NumCPUs is the total number of virtual processor cores to assign a vm.
 	//
 	// +optional
-	NumCPUs int32 `json:"cpus"`
+	NumCPUs int64 `json:"cpus"`
```

Comment:
Squash all the changes from this commit into the previous commit where these fields were originally defined.

Reply:
Okay.